demos/logging-stack/logging.cmake

```diff
@@ -0,0 +1,3 @@
+# Configuration for logging.
+set( LOGGING_INCLUDE_DIRS
```

Comment:
I would rather this be set in an appropriate `CMakeLists.txt`

Reply:
Yes, I agree. I plan on upgrading this to CMakeLists.txt in a follow-up PR.